packages/react-button/package.json

```diff
@@ -42,7 +42,9 @@
   },
   "dependencies": {
     "@fluentui/keyboard-key": "^0.3.1",
+    "@fluentui/react-icons-mdl2": "^1.1.1",
```

Comment:
I don't think we should be taking a dependency on the mdl2 icon package in converged

Reply:
Removed!